### ISSUE TITLE
OSX Sierra

### ISSUE THREAD

**Issue by nar53** [yujitach/MenuMeters#35]
Just for info, your version of menu meters is working on my OSX Sierra Hackintosh. However one minor glitch, my quad core cpu is showing as an 8 core cpu.

**Comment by myint** [comment]
I don't think that is a glitch. At least for me, I have a quad-core Mac Pro and it shows up properly as a 8 hyperthreaded cores.

https://en.wikipedia.org/wiki/Hyper-threading

**Comment by EricKerby** [comment]
Would it be difficult to provide an option to hide the hyper-threaded cores? I noticed iStat Menus 5 does offer that feature: "Hyper-Threading cores can now be hidden".

**Comment by yujitach** [comment]
not difficult, but I don't have the time.

**Comment by lukehamburg** [comment]
Yuji, is it "safe" to compile a current build from the master? I notice you just merged in all the PRs from @jthlim 

Thanks!

**Comment by yujitach** [comment]
@lukehamburg I went through @jthlim's changes line by line and they looked fine. In fact it seems it solved two issues #16 and #19 as a by-product. I'm running the latest on my machine and it looks fine too. I would appreciate if you could test it too.

**Comment by lukehamburg** [comment]
Hi Yuji, 

I tested both the 1.9.3 precompiled version as well as cloning the latest git source and compiling myself in Xcode 8.2 (8C38).  Both worked perfectly on macOS Sierra 10.12.2. Thank you for taking time to do this!  The only thing I noticed, is the version I compiled myself was 953,213 bytes vs. 1,117,494 bytes for the one on your website. I guess that is due to differences in Xcode version, compile flags or code signing.
